modules/ROOT/pages/queries/select-version.adoc

```diff
@@ -229,21 +230,21 @@ The table below outlines which Cypher version is assigned to databases in differ
 |===
 | Scenario | `db.query.default_language` | Databases
 
-| *Standard system upgrade to Neo4j 2025.03+*
+| *Standard DBMS upgrade to Neo4j 2025.03+*
 | Unset
-| *Existing system database:* `CYPHER 5` +
+| *Existing DBMS database:* `CYPHER 5` +
 *Existing user databases:* `CYPHER 5` +
 *New user databases default:* `CYPHER 5`
 
-| *Custom system upgrade to Neo4j 2025.03+*
-| Manually set to `CYPHER 25` by administrator
-| *Existing system database:* `CYPHER 5` +
+| *Custom DBMS upgrade to Neo4j 2025.03+*
+| Manually set to `CYPHER_25` by administrator
+| *Existing DBMS database:* `CYPHER 5` +
 *Existing user databases:* `CYPHER 5`  +
 *New user databases default:* `CYPHER 25`
 
 | *New installation of Neo4j 2025.03+* 
-| Set to `CYPHER 25`
-| *New system database:* `CYPHER 25` +
+| Set to `CYPHER_25`
+| *New DBMS database:* `CYPHER 25` +
```

Comment:
```suggestion
| *New system database:* `CYPHER 25` +
```
here as well, these point out what the system database in the DBMS gets as default